Phoniebox/rpc/PhonieboxRpcClient.py

```diff
@@ -25,8 +31,9 @@ def enqueue(self, request):
 
         try:
             server_response = self.queue.recv()
-        except:
-            print ("somethng went wrong")
+        except Exception as e:
+            print ("somethng went wrong:")
```

Comment:
I might be picky, but if we use something like "[ERROR] ...", it'll be easier for use to search and browser logs if required

Reply:
sure, error handling has not yet been focused on